utils.py

```diff
@@ -73,10 +75,34 @@ def gen_json(metadata_dict, metadata_name):
 
   filename = metadata_name + '.json'
   fileobj = open(filename, 'w')
-  fileobj.write(json.encode_pretty_printed_json(metadata_dict))
+  fileobj.write(canonicaljson.encode_pretty_printed_json(metadata_dict))
   fileobj.close()
 
 
+def json_to_dict(json_filepath):
```

Comment:
tuf/util.py has a similar function called load_json_file. Your function seems fine but tuf has more error checking in theirs. Take a look at their function and see if it may be more beneficial to just use their function or leverage some of their error checking in your function.